build/ci/Makefile

```diff
@@ -144,3 +144,15 @@ ci-e2e-delete-cluster: vault-gke-creds
     	-e "GKE_SERVICE_ACCOUNT_KEY_FILE=$(GO_MOUNT_PATH)/build/ci/$(GKE_CREDS_FILE)" \
     	cloud-on-k8s-ci-e2e \
     	bash -c "make -C operators set-context-gke delete-gke"
```

Comment:
IIUC, everytime we delete a gke cluster, the underlying disks are not deleted. This applies to e2e tests but also to our own clusters, right?
Should we do this everytime, as part of the `delete-gke` target?

Reply:
This fix applies to all disks, not only from clusters for e2e tests. No need to explicitly run `delete-gke`

Reply:
OK. But isn't there a bug in the `delete-gke` target (that we use ourselves) if the disks are not deleted?

Reply:
Yeah sure, if it runs and some unused disks are left, then it's a bug. I only run it before submitting original PR and it cleaned up everything at that moment.